### ISSUE TITLE
training question from ours dataset

### ISSUE THREAD

**Issue by kennyadelaide** [open-mmlab/mmdetection#10859]
I would like to ask why the training accuracy of mask rcnn model is very high when training data, but the evaluation accuracy is very low. I have done a lot of overfitting work, hoping to solve this problem, but the effect is still not good and the evaluation accuracy is still very low. After browsing and checking the data, there is no problem. What could possibly be the reason?

**Comment by hhaAndroid** [comment]
@kennyadelaide Can you provide more specific information?
